### PR TITLE
Various whitespace fixes

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

File: lua/entities/gmod_wire_expression2/core/custom/cl_streamcore.lua [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+E2Helper.Descriptions["streamVolume( nn )"] = "Changes the Nth stream's volume.\nVolume is between 0 and 1."
+E2Helper.Descriptions["streamRadius( nn )"] = "ONLY WORKS WITH 3D-DISABLED STREAMS!\nChanges the Nth stream's radius.\nRadius needs to be greater than 0.\nSee the streamMaxRadius function."
+E2Helper.Descriptions["streamHelp()"] = "Opens the StreamCore's workshop page which contains the full documentation using the Steam's Overlay and returns its URL."
+E2Helper.Descriptions["streamCanStart()"] = "Returns 1 if you are able to start a new stream."
+E2Helper.Descriptions["streamLimit()"] = "Returns the value of the streamcore_antispam_seconds server cvar."
+E2Helper.Descriptions["streamMaxRadius()"] = "Returns the value of the streamcore_maxradius server cvar."
+E2Helper.Descriptions["streamAdminOnly()"] = "Returns 1 if the StreamCore is admin-only.\nSee the streamcore_adminonly server cvar."
+E2Helper.Descriptions["streamDisable3D( n )"] = "If N > 0, the streams will use a \"fake\" 3D mode.\nThis is a workaround to obtain louder sounds.\nActive streams will not be changed."
```
Comment (brandonsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/cl_streamcore.lua]:
```suggestion
E2Helper.Descriptions["streamStart(e:ns)"] = "Streams a .mp3 sound from an URL at E's position.\nThis function has an antispam, see the streamCanStart function.\nThe default volume is set to 1."
E2Helper.Descriptions["streamStart(e:nns)"] = "Streams a .mp3 sound from an URL at E's position.\nThis function has an antispam, see the streamCanStart function."
E2Helper.Descriptions["streamStart(e:nsn)"] = "Streams a .mp3 sound from an URL at E's position.\nThis function has an antispam, see the streamCanStart function."
E2Helper.Descriptions["streamStop(n)"] = "Stops the Nth stream started earlier using the streamStart function."
E2Helper.Descriptions["streamVolume(nn)"] = "Changes the Nth stream's volume.\nVolume is between 0 and 1."
E2Helper.Descriptions["streamRadius(nn)"] = "ONLY WORKS WITH 3D-DISABLED STREAMS!\nChanges the Nth stream's radius.\nRadius needs to be greater than 0.\nSee the streamMaxRadius function."
E2Helper.Descriptions["streamHelp()"] = "Opens the StreamCore's workshop page which contains the full documentation using the Steam's Overlay and returns its URL."
E2Helper.Descriptions["streamCanStart()"] = "Returns 1 if you are able to start a new stream."
E2Helper.Descriptions["streamLimit()"] = "Returns the value of the streamcore_antispam_seconds server cvar."
E2Helper.Descriptions["streamMaxRadius()"] = "Returns the value of the streamcore_maxradius server cvar."
E2Helper.Descriptions["streamAdminOnly()"] = "Returns 1 if the StreamCore is admin-only.\nSee the streamcore_adminonly server cvar."
E2Helper.Descriptions["streamDisable3D(n)"] = "If N > 0, the streams will use a \"fake\" 3D mode.\nThis is a workaround to obtain louder sounds.\nActive streams will not be changed." 
```

File: lua/entities/gmod_wire_expression2/core/custom/streamcore.lua [hunk 2]
```diff
@@ -1,153 +1,153 @@
...
+e2function void streamDisable3D( disable )
+    self.data = self.data or {}
+    self.data.no3d = ( disable != 0 )
+end
+
+__e2setcost( 5 )
+e2function string streamHelp()
+    net.Start( "XTS_SC_StreamHelp" )
+    net.Send( self.player )
+    return "http://steamcommunity.com/sharedfiles/filedetails/?id = 442653157"
```
Comment (sarahsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/streamcore.lua]:
```suggestion
    return "http://steamcommunity.com/sharedfiles/filedetails/?id=442653157"
```

File: lua/entities/gmod_wire_expression2/core/custom/streamcore.lua [hunk 3]
```diff
@@ -1,153 +1,153 @@
...
+        net.WriteEntity( owner )
+        net.WriteBool( no3d )
+        net.WriteFloat( radius )
+    net.Broadcast()
+end
+
+__e2setcost( 1 )
+e2function void streamDisable3D( disable )
+    self.data = self.data or {}
+    self.data.no3d = ( disable != 0 )
```
Comment (sarahsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/streamcore.lua]:
```suggestion
    self.data.no3d = disable ~= 0
```

File: lua/entities/gmod_wire_expression2/core/custom/streamcore.lua [hunk 4]
```diff
@@ -1,153 +1,153 @@
...
+    end
+end
+
+e2function void streamRadius( id, radius )
+    local index = self.entity:EntIndex() .. "-" .. id
+    local maxradius = GetConVarNumber( "streamcore_maxradius" )
+    radius = math.Clamp( radius, 0, maxradius )
+    local streamtbl = streams[index]
+    if not streamtbl then return end
+    if radius != streamtbl[3] then
```
Comment (sarahsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/streamcore.lua]:
```suggestion
    if radius ~= streamtbl[3] then
```

File: lua/entities/gmod_wire_expression2/core/custom/streamcore.lua [hunk 5]
```diff
@@ -1,153 +1,153 @@
...
+    end
+end
+
+__e2setcost( 15 )
+e2function void streamVolume( id, volume )
+    local index = self.entity:EntIndex() .. "-" .. id
+    volume = math.Clamp( volume, 0, 1 )
+    local streamtbl = streams[index]
+    if not streamtbl then return end
+    if volume != streamtbl[2] then
```
Comment (sarahsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/streamcore.lua]:
```suggestion
    if volume ~= streamtbl[2] then
```

File: lua/autorun/cl_streamcore.lua [hunk 6]
```diff
@@ -1,111 +1,111 @@
...
+    for index, streamtbl in pairs( streams ) do
+        if streamStop( index ) then
+            print( "[StreamCore] Stream #" .. index .. " successfully stopped!" )
+        end
+    end
+    print( "[StreamCore] Purge done." )
+end )
+
+net.Receive( "XTS_SC_StreamHelp", function( len )
+    gui.OpenURL( "http://steamcommunity.com/sharedfiles/filedetails/?id = 442653157" )
```
Comment (plally) [inline lua/autorun/cl_streamcore.lua]:
```suggestion
    gui.OpenURL( "http://steamcommunity.com/sharedfiles/filedetails/?id=442653157" )
```

**Pull request merged**